projects/packages/forms/src/blocks/field-file/index.js

```diff
@@ -0,0 +1,45 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { upload } from '@wordpress/icons';
+import { getIconColor } from '../contact-form/util/block-icons';
+import defaultSettings from '../shared/settings';
+import edit from './edit';
+import save from './save';
+
+const name = 'field-file';
+const settings = {
+	...defaultSettings,
+	title: __( 'File upload field', 'jetpack-forms' ),
```

Comment:
This uses sentence case, whereas the other fields use title case. Should this be the following?

```suggestion
	title: __( 'File Upload Field', 'jetpack-forms' ),
```

Reply:
Our style guide is to use sentence case (pgijrj-42-p2, PCYsg-5n-p2) and we've been doing new UIs with that guideline. Now with refactor behind, we can follow-up with old UIs as well and I'm updating the rest of field names here:
- https://github.com/Automattic/jetpack/pull/43818